examples/slack-connect-example/pages/index.tsx

```diff
@@ -39,124 +41,104 @@ export default function Home() {
   }
 
   return (
-    <>
-      <div
-        style={{
-          marginBottom: "20px",
-          fontFamily: "monospace",
-          fontSize: "40px",
-        }}
+    <KnockProvider
+      apiKey={process.env.NEXT_PUBLIC_KNOCK_CLIENT_ID!}
+      userId={user.id}
+      host={process.env.NEXT_PUBLIC_KNOCK_API_URL}
+      userToken={localStorage.getItem("knock-user-token")!}
+    >
+      <KnockSlackProvider
+        knockSlackChannelId={process.env.NEXT_PUBLIC_KNOCK_SLACK_CHANNEL_ID!}
+        tenant={tenant}
```

Comment:
I think that we may not be shipping this example app as is because jeff is working on a separate one to showcase this work - if we do end up keeping this one I can throw if they don't exist